docs/index.md

```diff
@@ -6,11 +6,38 @@ hide:
 ---
 
 # Discordoo Documentation
-Hi there! This is Discordoo documentation. Here you will learn all the basics and internals of Discordoo.
+Welcome! This is documentation of Discordoo API in English language.
 
-## Where to start?
-If you are a beginner, go to the `Base guide` tab. This is the friendliest place where you can learn how to work with Discordoo.
-Or you can go to the `Api` tab, it contains a precise description for everything in Discordoo, but it wasn't intended as a good place to start.
+## **Important!**
+Discordoo and this library are intended to be used by people, who know what they do, can write/read some basic TypeScript/JavaScript and know how to work with Node.js runtime.
+If you are a beginner, you should learn language and only after start learning Discordoo.
 
-## Important notice
-This documentation is intended to be read by people, who know TypeScript and Node.js on beginner level. We will not help you if you don't know how to use Node.js or write basic TypeScript code.
+Of course, you may not learn TS/JS, however, you will not get any support on official support server.
```

Comment:
```suggestion
Of course, you may not learn TS/JS, however, but you will not get any support on official support server.
```

Reply:
it sounds kinda awkward, however and but are not used together

Reply:
oh, sorry, I didn't see that there is however